typings/Application.d.ts

```diff
@@ -21,7 +21,17 @@ export type ApplicationConfig = {
 export class Application {
   constructor(config: ApplicationConfig);
   validateListeners(): void;
-  state: StateManager;
+  state: {
+    ready: Property<boolean>,
+    pause: Property<boolean>,
+    captionsMuted: Property<boolean>,
+    playOptions: Property<object>,
+    soundVolume: Property<number>,
+    musicVolume: Property<number>,
+    voVolume: Property<number>,
+    sfxVolume: Property<number>,
+    [key:string]: Property
```

Comment:
@SmithAF, maybe this should be `[key:string]: Property<any>`? VSCode doesn't like this as-is, is treating it as just `any`.